extras/util/validation.js

```diff
@@ -0,0 +1,53 @@
+const users = [
+    {
+        traineeEmail: 'trainee1@successive.tech',
+        reviewerEmail: 'reviewer1@successive.tech',
+    },
+    {
+        traineeEmail: 'trainee2@succedfssive.tech',
+        reviewerEmail: 'reviewer$ff2@successive.tech',
+    },
+    {
+        traineeEmail: 'trainee3@successive.tech',
+        reviewerEmail: 'reviewer3@successive.tech',
+    },
+    {
+        traineeEmail: 'trainee4@successive.tec',
+        reviewerEmail: 'reviewer4@123successive.tech',
+    },
+    {
+        traineeEmail: 'trainee5@successive.tech',
+        reviewerEmail: 'reviewer5@successive.tech',
+    }
+    ]
+function validateEmail(email){
```

Comment:
@pvvarun Add a condition if there is no email, return false OR you can just put default value of email as blank string i.e., "". This is just to make your program foolproof